delfin/drivers/dell_emc/unity/rest_handler.py

```diff
@@ -167,7 +172,16 @@ def get_all_alerts(self, page_number):
                                  'messageId,message,description,'
                                  'descriptionId,state',
                                  page_number)
-        result_json = self.get_rest_info(url)
+        result_json = self.get_rest_info(url, consts.ALERT_TIMEOUT)
+        return result_json
+
+    def get_all_alerts_without_state(self, page_number):
+        url = '%s?%s&page=%s' % (RestHandler.REST_ALERTS_URL,
+                                 'fields=id,timestamp,severity,component,'
+                                 'messageId,message,description,'
+                                 'descriptionId',
+                                 page_number)
+        result_json = self.get_rest_info(url, consts.ALERT_TIMEOUT)
```

Comment:
The second parameter of get_rest_info() is data, not calltimeout.

Reply:
done